modules/local/seqera_runs_dump/main.nf

```diff
@@ -18,11 +18,11 @@ process SEQERA_RUNS_DUMP {
     script:
     def args = task.ext.args ?: ''
     def args2 = task.ext.args2 ?: ''
-    prefix = task.ext.prefix ?: "${meta.id}"
-    metaOut = meta + getRunMetadata(meta, log, api_endpoint, java_truststore_path, java_truststore_password)
-    fusion = metaOut.fusion ? '--add-fusion-logs' : ''
-    javaTrustStore = java_truststore_path ? "-Djavax.net.ssl.trustStore=${java_truststore_path}" : ''
-    javaTrustStorePassword = java_truststore_password ? "-Djavax.net.ssl.trustStorePassword=${java_truststore_password}" : ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def metaOut = meta + getRunMetadata(meta, log, api_endpoint, java_truststore_path, java_truststore_password)
```

Comment:
```suggestion
    prefix = task.ext.prefix ?: "${meta.id}"
    metaOut = meta + getRunMetadata(meta, log, api_endpoint, java_truststore_path, java_truststore_password)
```

Reply:
These variables are used in the `output` section so we can't scope with `def`? Be good to test this if we haven't already.

Reply:
Why remove the `def`s here, Harshil? The LSP recommends adding them. Do you think adding in the def statements will break something?

Reply:
Ah, ok. Let me test.